packages/syntax/src/Syntax.tsx

```diff
@@ -1,14 +1,48 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
-import { css } from '@leafygreen-ui/emotion';
+import flatten from 'lodash/flatten';
+import { cx, css } from '@leafygreen-ui/emotion';
+import { fontFamilies } from '@leafygreen-ui/tokens';
 // Import from core so we can register the appropriate languages ourselves
 import hljs from 'highlight.js/lib/core';
 import hljsDefineGraphQL from 'highlightjs-graphql';
-import CodeWrapper from './CodeWrapper';
-import { Language, SyntaxProps } from './types';
+import { Language, SyntaxProps, Mode } from './types';
 import { SupportedLanguages, languageParsers } from './languages';
 import { injectGlobalStyles } from './globalStyles';
-import renderingPlugin from './renderingPlugin';
+import renderingPlugin, { TableContent } from './renderingPlugin';
+import { SyntaxContext } from './SyntaxContext';
+
+export function expandRangeTuple(
+  tuple: [number, number],
```

Comment:
can this also take in min/max arguments? the max we pass in should just be however many lines of code there are. don't want the window to freeze up when someone passes in `[0, Infinity]`.

Reply:
Passing `Infinity` seems like an almost impossible edge case, but I'll do something to make sure the tab doesn't freeze in the off chance that happens. Not in love with the min/max args.